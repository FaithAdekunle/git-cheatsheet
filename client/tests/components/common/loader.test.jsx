import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Loader from '../../../src/app/components/common/loader';

describe('<Loader />', () => {
  test('should match snapshot', () => {
    const wrapper = shallow(<Loader />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
