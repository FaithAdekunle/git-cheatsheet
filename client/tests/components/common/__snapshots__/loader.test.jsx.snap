// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loader /> should match snapshot 1`] = `
<div
  className="common"
>
  <ModalOverlay />
  <div
    className="fixed loader-modal"
  >
    <div
      className="spinner"
    >
      <FontAwesomeIcon
        border={false}
        className="fa-spin"
        fixedWidth={false}
        flip={null}
        icon="spinner"
        inverse={false}
        listItem={false}
        mask={null}
        pull={null}
        pulse={false}
        rotation={null}
        size={null}
        spin={false}
        symbol={false}
        title=""
        transform={null}
      />
    </div>
  </div>
</div>
`;
